src/components/FormAlertWrapper.tsx

```diff
@@ -48,11 +45,11 @@ function FormAlertWrapper({
     isAlertVisible = false,
     isMessageHtml = false,
     message = '',
-    network,
     onFixTheErrorsLinkPressed = () => {},
 }: FormAlertWrapperProps) {
     const styles = useThemeStyles();
     const {translate} = useLocalize();
+    const [network] = useOnyx(ONYXKEYS.NETWORK, {canBeMissing: true});
```

Comment:
Why not use `@hooks/useNetwork` to consume the context?

Reply:
Sure! I didn't know it is preferred 👌